src/test/scala/cromwell/engine/backend/CromwellRuntimeAttributeSpec.scala

```diff
@@ -1,72 +1,68 @@
 package cromwell.engine.backend
 
+import java.util.UUID
+
 import com.google.api.services.genomics.model.Disk
 import cromwell.engine.backend.runtimeattributes.{AttributeMap, CromwellRuntimeAttributes, ContinueOnReturnCodeFlag, ContinueOnReturnCodeSet}
 import cromwell.engine.workflow.WorkflowOptions
 import wdl4s.NamespaceWithWorkflow
+import cromwell.CromwellTestkitSpec.TestWorkflowManagerSystem
 import cromwell.engine.backend.CromwellRuntimeAttributeSpec._
+import cromwell.engine.backend.jes.JesBackend
+import cromwell.engine.backend.local.LocalBackend
+import cromwell.engine.backend.runtimeattributes._
+import cromwell.engine.workflow.BackendCallKey
+import cromwell.engine.{WorkflowDescriptor, WorkflowId}
+import cromwell.util.SampleWdl
+import org.scalatest.prop.TableDrivenPropertyChecks._
+import org.scalatest.prop.Tables.Table
 import org.scalatest.{EitherValues, FlatSpec, Matchers}
 
 import scala.util.{Failure, Success}
 
 object CromwellRuntimeAttributeSpec {
-  val DockerValuePlaceholder = "DOCKERVALUEPLACEHOLDER"
-  val WdlValueForDockerAttribute = "ubuntu:latest"
-  val WorkflowWithRuntime =
-    """
-      |task ps {
-      |  command {
-      |    ps
-      |  }
-      |  output {
-      |    File procs = stdout()
-      |  }
-      |  runtime {
-      |    docker: "DOCKERVALUEPLACEHOLDER"
-      |  }
-      |}
-      |
-      |task cgrep {
-      |  String pattern
-      |  File in_file
-      |  command {
-      |    grep '${pattern}' ${in_file} | wc -l
-      |  }
-      |  output {
-      |    Int count = read_int(stdout())
-      |  }
-      |  runtime {
-      |    docker: "DOCKERVALUEPLACEHOLDER"
-      |  }
-      |}
-      |
-      |task wc {
-      |  File in_file
-      |  command {
-      |    cat ${in_file} | wc -l
-      |  }
-      |  output {
-      |    Int count = read_int(stdout())
-      |  }
-      |  runtime {
-      |     docker: "DOCKERVALUEPLACEHOLDER"
-      |  }
-      |}
-      |
-      |workflow three_step {
-      |  call ps
-      |  call cgrep {
-      |    input: in_file=ps.procs
-      |  }
-      |  call wc {
-      |    input: in_file=ps.procs
-      |  }
-      |}
-      |
-    """.stripMargin.replace(DockerValuePlaceholder, WdlValueForDockerAttribute)
 
-  val WorkflowWithoutRuntime =
-    """
+  object WorkflowWithStaticRuntime extends SampleWdl {
```

Comment:
`SampleWdl` is ~1700 lines of code now and moving these over there will be 2000+ lines of code... I thought they might be better here since they'll likely never be used elsewhere.  Any opinions @Horneth or @geoffjentry?